app/server/api/credit_transfer_api.py

```diff
@@ -473,12 +473,25 @@ def post(self):
                     'message': 'No existing users involved in this transfer',
                     'data': {}
                 }
-            # Case 3: Two non-Sempo users who have both interacted with Sempo users before transact with one another
+            # Case 3: Two non-Sempo users, at least one of whom has interacted with Sempo users before transacting with one another
             # We don't have to track this either!
-            elif (maybe_recipient_transfer_account
-                  and maybe_recipient_transfer_account.account_type == TransferAccountType.EXTERNAL
-                  and maybe_sender_transfer_account
-                  and maybe_sender_transfer_account.account_type == TransferAccountType.EXTERNAL):
+            elif (
```

Comment:
I did a little clean up here just because i had trouble reading it past the 121 char line limit. Probably worth keeping in mind for long stuff like this